include/libdnf/comps/comps.hpp

```diff
@@ -27,19 +27,29 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include "group/group.hpp"
 #include "group/sack.hpp"
 
+extern "C" {
+#include <solv/pool.h>
+#include <solv/repo.h>
+}
 
 namespace libdnf::comps {
 
 
 class Comps {
 public:
+    explicit Comps() { pool = pool_create(); }
```

Comment:
The pool should be part of a sack (Sack is libdnf's wrapper of libsolv's pool for RPM packages).
Although this might be different because I can imagine that groups and environments will share the same pool.
In such the pool must be either in Comps as it is already or in GroupSack and EnvironmentSack would only contain a pointer to GroupSack's pool.